ffi/capi/bindings/js/ICU4XCustomTimeZone.d.ts

```diff
@@ -64,58 +64,53 @@ export class ICU4XCustomTimeZone {
 
    * Returns the value of the `gmt_offset` field as offset seconds.
 
-   * Errors if the `gmt_offset` field is empty.
+   * Returns null if the `gmt_offset` field is empty.
```

Comment:
Observation: the docs say "returns null" but in TS it returns `undefined`

Reply:
Yeah docs say different things, it's hard to write truly language-agnostic docs.